File: flutter-prj/crud_api_http_methods/lib/model/produto.dart

class Produto {
  int _id;
  String _descricao;
  int _qtd;
  double _valor;

  Produto(int id, String descricao, int qtd, double valor) {
    this._id = id;
    this._descricao = descricao;
    this._qtd = qtd;
    this._valor = valor;
  }

  Produto.fromGet(String descricao, int qtd, double valor) {
    this._descricao = descricao;
    this._qtd = qtd;
    this._valor = valor;
  }

  int get id {
    return _id;
  }

  String get descricao {
    return _descricao;
  }

  int get qtd {
    return _qtd;
  }

  double get valor {
    return _valor;
  }

}

File: spring-crud-api/crud-api-flutter-requests/src/main/java/com/crud/domain/Produto.java
package com.crud.domain;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;

import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonInclude.Include;

import lombok.Data;
import lombok.NoArgsConstructor;

@JsonInclude(Include.ALWAYS)
@Entity
@Table(name="tbl_produtos")
@Data
@NoArgsConstructor
public class Produto {
	
	@Id 
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	@Column(name="id")
	private Integer id;
	@Column(name="descricao")
	private String descricao;
	@Column(name="qtd")
	private int qtd;
	@Column(name="valor")
	private Double valor;

}
